spec/factories/ext_management_system.rb

```diff
@@ -275,6 +275,20 @@
     authtype { "default" }
   end
 
+  factory :ems_azure_stack,
```

Comment:
I believe these can live in your plugin and don't need to be here.

Reply:
Moved as you suggest, thanks for pointing out.